File: week1/pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get height from user in the range <1,8>
    static int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);

    // Print a pyramid with the provided height
    for (int i=0; i < height; i++)
    {
        for (int j=0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        for (int j=0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/pset6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height <= 8 and height >= 1:
        for i in range(height):
            print(" " * (height - i - 1), end="")
            print("#" * (i + 1))
        break

